operator/apis/loki/v1/lokistack_types.go

```diff
@@ -1216,11 +1216,11 @@ const (
 	// Instead, they are generated during runtime using a service, which allows for shorter-lived credentials and
 	// much more granular control. This authentication mode is not supported for all object storage types.
 	CredentialModeToken CredentialMode = "token"
-	// CredentialModeManaged represents the usage of short-lived tokens retrieved from a credential source.
+	// CredentialModeTokenCCO represents the usage of short-lived tokens retrieved from a credential source.
 	// This mode is similar to CredentialModeToken,but instead of having a user-configured credential source,
 	// it is configured by the environment, for example the Cloud Credential Operator in OpenShift.
 	// This mode is only supported for certain object storage types in certain runtime environments.
```

Comment:
I think we can be less vague in the comment now:

```suggestion
	// CredentialModeTokenCCO represents the usage of short-lived tokens retrieved from a credential source.
	// This mode is similar to CredentialModeToken, but instead of having a user-configured credential source,
	// it is configured by the environment and the operator relies on the Cloud Credential Operator to provide
	// a secret. This mode is only supported for certain object storage types in certain runtime environments.
```